pkg/clusteragent/README.md

```diff
@@ -0,0 +1,80 @@
+# Datadog Cluster Agent - DCA
+
+[![CircleCI](https://circleci.com/gh/DataDog/datadog-agent/tree/master.svg?style=svg&circle-token=dbcee3f02b9c3fe5f142bfc5ecb735fdec34b643)](https://circleci.com/gh/DataDog/datadog-agent/tree/master)
+[![Build status](https://ci.appveyor.com/api/projects/status/kcwhmlsc0oq3m49p/branch/master?svg=true)](https://ci.appveyor.com/project/Datadog/datadog-agent/branch/master)
+[![GoDoc](https://godoc.org/github.com/DataDog/datadog-agent?status.svg)](https://godoc.org/github.com/DataDog/datadog-agent)
+
+The Datadog Cluster Agent (referred to as DCA) faithfully collects events and metrics and brings them to
+[Datadog](https://app.datadoghq.com) on your behalf so that you can do something
+useful with your monitoring and performance data.
```

Comment:
I think an explanation about the pressure on the API server without the DCA on big cluster would be nice